cache/web3/ipfs.json

```diff
@@ -4,48 +4,94 @@
     "title": "Aave Pool update",
     "author": "BGD Labs @bgdlabs",
-    "discussions": "https://governance.aave.com/t/pre-cautionary-measures-on-three-aave-v3-assets/16037"
+    "discussions": "https://governance.aave.com/t/pre-cautionary-measures-on-three-aave-v3-assets/16037",
+    "originalIpfsHash": "0xf92585de2ff0761e3d86d0e3253d2f49b2a916ef63784834f5e7230efb1bd194"
```

Comment:
why the addition - it's the index of the object already (if you access it you already know it)